src/services/challenges.js

```diff
@@ -160,6 +160,14 @@ export function patchChallenge (challengeId, params) {
   })
 }
 
+/*
+* Deletes the challenge with the provided id.
+* @param challengeId
+*/
+export function deleteChallenge (challengeId) {
+  return axiosInstance.delete(`${CHALLENGE_API_URL}/${challengeId}`).then(rs => rs)
```

Comment:
@mark-nakachon it feels like `.then(rs => rs)` doesn't do anything and can be removed.